azuredevops.go

```diff
@@ -74,30 +74,36 @@ func (m importModule) runAzureDevOpsHandler(c *gin.Context) {
 		return
 	}
 
-	importer := importer.NewAzureImporter(c, &client)
-	token := wharfapi.Token{
-		TokenID:  i.TokenID,
-		Token:    i.Token,
-		UserName: i.UserName}
-	provider := wharfapi.Provider{
-		ProviderID: i.ProviderID,
-		Name:       providerName,
-		URL:        i.URL,
-		TokenID:    i.TokenID}
+	imp := importer.NewAzureImporter(c, &client)
```

Comment:
Im not particularly fond of this refactor it goes from an semi-informative name to an uncommon abreviation that in itself is uninformative of what it represents.

Reply:
Was able to keep the name as `importer` by ordering the code differently 👍🏻